### ISSUE TITLE
Select lists cannot be operated from the keyboard if they have an onchange handler that performs navigation, because the handler fires as the user moves the selection up and down using the keyboard

### ISSUE THREAD

**Issue by Horlix** [canonical-web-and-design/ubuntu.com#11892]
Moving through list items using the arrow keys will jump to another page unexpectedly.

Example URLs:
- https://ubuntu.com/security/certifications/docs
- https://ubuntu.com/security/certifications/docs/16-18/cc
- https://ubuntu.com/security/certifications/docs/16-18/cis

`<select name="version-select" id="version-select" onchange="window.location.href=this.value">`

**Comment by Horlix** [comment]
Full set of URLs

[select lists cannot be operated from the keyboard if they have an onchange handler that performs navigation, because the handler fires as the user moves the selection up and down using the keyboard.csv](https://github.com/canonical-web-and-design/ubuntu.com/files/9213984/select.lists.cannot.be.operated.from.the.keyboard.if.they.have.an.onchange.handler.that.performs.navigation.because.the.handler.fires.as.the.user.moves.the.selection.up.and.down.using.the.keyboard.csv)
